project-base/src/Shopsys/ShopBundle/DataFixtures/Demo/OrderStatusDataFixture.php

```diff
@@ -51,6 +70,24 @@ protected function createOrderStatusReference(
         $referenceName
     ) {
         $orderStatus = $this->orderStatusFacade->getById($orderStatusId);
+        $orderStatusData = $this->orderStatusDataFactory->createFromOrderStatus($orderStatus);
+        foreach ($this->domain->getAllLocales() as $locale) {
+            switch ($referenceName) {
+                case self::ORDER_STATUS_NEW:
+                    $orderStatusData->name[$locale] = t('New [adjective]', [], 'dataFixtures', $locale);
+                    break;
+                case self::ORDER_STATUS_IN_PROGRESS:
+                    $orderStatusData->name[$locale] = t('In Progress', [], 'dataFixtures', $locale);
+                    break;
+                case self::ORDER_STATUS_DONE:
+                    $orderStatusData->name[$locale] = t('Done', [], 'dataFixtures', $locale);
+                    break;
+                case self::ORDER_STATUS_CANCELED:
+                    $orderStatusData->name[$locale] = t('Canceled', [], 'dataFixtures', $locale);
+                    break;
+            }
```

Comment:
Throw an exception when switch does not match to any case?

Reply:
ok, fixed in 11128d49a